src/fido/types.h

```diff
@@ -145,6 +145,7 @@ typedef struct fido_cred {
 	fido_attcred_t    attcred;       /* returned credential (key + id) */
 	fido_attstmt_t    attstmt;       /* attestation statement (x509 + sig) */
 	fido_blob_t       largeblob_key; /* decoded large blob key */
+	fido_blob_t       blob;          /* blob to store with the credBlob extension */
```

Comment:
s/blob to store with the credBlob extension/FIDO 2.1 credBlob/, maybe?

Reply:
fixed in ff5b0b66b67aa6f336f84db06a3c20ddd87872fc